gateway/1.7/modules/ROOT/pages/flex-review-prerequisites.adoc

```diff
@@ -135,4 +135,4 @@ Port 9998 is reserved for internal processes, and should not be used in `ApiInst
 [[flex-gateway-hyperforce]]
 == Flex Gateway on Hyperforce
 
-Hyperforce supports Flex Gateway versions 1.7 and later. Because Flex Gateway running in Local Mode is completely self-hosted, it is not affected by Hyperforce. For Flex Gateway running in Connected Mode, some API Manager Hyperforce limitations affect Mule Gateway. To learn more, see xref:api-manager::latest-overview-concept.adoc#APIM-hyperforce[API Manager on Hyperforce].
\ No newline at end of file
+Hyperforce supports Flex Gateway versions 1.7 and later. Because Flex Gateway running in Local Mode is completely self-hosted, isn't not affected by Hyperforce. For Flex Gateway running in Connected Mode, some API Manager Hyperforce limitations affect Mule Gateway. To learn more, see xref:api-manager::latest-overview-concept.adoc#APIM-hyperforce[API Manager on Hyperforce].
```

Comment:
```suggestion
Hyperforce supports Flex Gateway versions 1.7 and later. Because Flex Gateway running in Local Mode is completely self-hosted, it isn't affected by Hyperforce. For Flex Gateway running in Connected Mode, some API Manager Hyperforce limitations affect Mule Gateway. To learn more, see xref:api-manager::latest-overview-concept.adoc#APIM-hyperforce[API Manager on Hyperforce].
```